Sources/Swift/Integrations/SessionReplay/SentryOnDemandReplay.swift

```diff
@@ -112,17 +121,17 @@ class SentryOnDemandReplay: NSObject {
         videoWriter.startSession(atSourceTime: .zero)
         
         var frameCount = 0
-        let (frames, start, end) = filterFrames(beginning: beginning, end: beginning.addingTimeInterval(duration))
+        let (framesPath, start, end) = filterFrames(beginning: beginning, end: beginning.addingTimeInterval(duration))
```

Comment:
`l`: This name better reflects that it's an array.
```suggestion
        let (framePaths, start, end) = filterFrames(beginning: beginning, end: beginning.addingTimeInterval(duration))
```